src/mpc-hc/PPageSheet.cpp

```diff
@@ -354,6 +354,30 @@ LRESULT CPPageSheet::OnDpiChanged(WPARAM wParam, LPARAM lParam) {
     return FALSE;
 }
 
+int CALLBACK PropSheetCallBackRTL(HWND hWnd, UINT message, LPARAM lParam) {
+    switch (message) {
+        case PSCB_PRECREATE:
+        {
+            //arabic or hebrew
+            if (AfxGetAppSettings().language == 1025 || AfxGetAppSettings().language == 1037) {

```

Comment:
Maybe add IsLangRTL() helper function?

Reply:
Good idea, but the translations code did it a different way.

```
if (PRIMARYLANGID(languageResource.localeID) == LANG_ARABIC || PRIMARYLANGID(languageResource.localeID) == LANG_HEBREW)
```

I didn't have that option so I did it a different way.

Reply:
One other issue--by the time SetLanguage() is called, appsettings is not updated (see PPagePlayer.cpp lin 186-191).  Need to redo that, too.